README.md

```diff
@@ -41,25 +41,25 @@ and convert them to Brigade events that are emitted into Brigade's event bus.
 * [GitHub Gateway](https://github.com/brigadecore/brigade-github-gateway)
 * [Slack Gateway](https://github.com/brigadecore/brigade-slack-gateway)
 
+### Other Event Sources
+
+* [Cron-Event-Source](https://github.com/brigadecore/brigade-cron-event-source)
+* [Brigade Noisy Neighbor](https://github.com/brigadecore/brigade-noisy-neighbor)
+  (The Brigade team is utilizing "Brigade Noise Neighbor" to keep our own internal 
+  Brigade installation under a steady load. We hope the larger event volumes than 
+  what we generate on our own will help us to identify and resolve bugs sooner.)
```

Comment:
@hamees-sayed let's drop this text while we're at it.

Reply:
Yup definitely, I remove the dash and this text